stable/prometheus-operator/Chart.yaml

```diff
@@ -11,7 +11,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 5.16.0
+version: 5.16.1
```

Comment:
Because this is a feature, rather than a bugfix, you should bump the minor version in semver

Reply:
True, thanks, bumped the version